### DISCUSSION TITLE
Proxy, does it work?

### DISCUSSION THREAD

**Discussion by Einstein-here** [pear-devs/pear-desktop#691] [Q&A]
Have tried everything, scoured through loads of issues relating to proxies went and used proxies from different sources but nothing seems to be working all it displays is an error (please refer to the image down below. 

At first, I speculated that maybe it's just a youtube thing, but no when trying to access YT music through a VPN, I'm able to access it and works flawlessly, but since I have to use another sort of program that requires a good connection like gaming perhaps, I can't choose a VPN since my connection would just be everywhere.

I hope I'm doing something wrong, would like to know a solution to this, unless it's just a sort of bug that is still undergoing review.

![image](https://user-images.githubusercontent.com/78808392/163272213-134e427a-163b-4601-9941-e58707a53764.png)
![image](https://user-images.githubusercontent.com/78808392/163271870-6c75350a-cdf0-494f-8337-7ed861a6817e.png)

**Comment by Araxeus** [comment]
are you completely sure that the address you are providing is valid? should look something like `socks5://127.0.0.1:1080`
you can test if your proxy works using curl:
`curl -x socks5://127.0.0.1:1080 http://example.com/`

the app just set a commandline switch which is handled by chromium
https://github.com/th-ch/youtube-music/blob/57ec0a463de871622f0d4c1237477c605e490275/index.js#L65
https://www.chromium.org/developers/design-documents/network-stack/socks-proxy/



<details>
<summary> you can also check that its saved well in config.json</summary>

<br>* Windows: %AppData%/Youtube Music (MyUser/AppData/Roaming/Youtube Music)
<br>* Linux: ~/.config/Youtube Music
<br>* MacOS: ~/Library/Application\ Support/YouTube\ Music/config.json

</details>
(just a precaution, there shouldn't be a problem there)
